tests/integration/records/serializers/json/test_authors.py

```diff
@@ -0,0 +1,332 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2019 CERN.
+#
+# inspirehep is free software; you can redistribute it and/or modify it under
+# the terms of the MIT License; see LICENSE file for more details.
+
+import json
+
+from helpers.providers.faker import faker
+from invenio_accounts.testutils import login_user_via_session
+
+from inspirehep.accounts.roles import Roles
+
+
+def test_literature_authors_json(api_client, db, create_record):
```

Comment:
shouldn't this be part of ``test_literatrue.py``?

Reply:
I agree, (copy paste)